dbms/src/Flash/Planner/PhysicalPlanNode.h

```diff
@@ -62,9 +62,9 @@ class PhysicalPlanNode
     /// Obtain a sample block that contains the names and types of result columns.
     virtual const Block & getSampleBlock() const = 0;
 
-    bool isRecordProfileStreams() const { return is_record_profile_streams; }
+    bool isTiDBOperator() const { return is_tidb_operator; }
 
-    void disableRecordProfileStreams() { is_record_profile_streams = false; }
+    void notATiDBOperator() { is_tidb_operator = false; }
```

Comment:
Should use `isTiDBOperator/notTiDBOperator` or `isATiDBOperator/notATiDBOperator`, not the mixed ones :(